docker/cortx-deploy/third-party-rpms.txt

```diff
@@ -20,5 +20,7 @@ python36-pika-0.10.0 #cortx-s3srever
 hiredis-0.12.1 #cortx-s3srever
 sshpass
 sudo
+file
```

Comment:
Please add component name infront of package name. 
```
file #cortx-motr
less #cortx-motr
```

Reply:
incorporated